docs/runjobs/webui/jupyter.md

```diff
@@ -31,9 +32,9 @@ In the basic settings, the following settings are available:
 
 In the advanced settings, you can find the following settings:
 
-- **Custom init _Text_ / Script to start:** Enter the full script for configuring up the environment to run Jupyter lab in the text box.
-- **Custon init _File_ / Path to script:** Enter the path to the script for configuring the environment to run Jupyter lab in the text box.
+- **Custom init _Text_ / Script to start:** Enter the full script for configuring up the environment to run JupyterLab in the text box.
+- **Custom init _File_ / Path to script:** Enter the path to the script for configuring the environment to run JupyterLab in the text box.
 
-The script or path to script will be sourced by bash before starting Jupyter.
+Bash will `source` the script or path to script before starting Jupyter.
```

Comment:
```suggestion
Bash will source the script or path to script before starting Jupyter.
```

Reply:
This was an intentional change, since `source` is a Bash command, it would be easier for a reader to google what it means (c.f. executing a script in a subinterpreter).